scripts/stubtest/allowlist.txt

```diff
@@ -27,6 +27,13 @@ django.core.files.storage.default_storage
 django.contrib.admin.models.LogEntry_RelatedManager
 django.contrib.auth.models.Permission_RelatedManager
 
+# '<Model>_ManyRelatedManager' entries are plugin generated and these subclasses only exist
+# _locally/dynamically_ runtime -- Created via
+# 'django.db.models.fields.related_descriptors.create_forward_many_to_many_manager'
+django.contrib.auth.models.Group_ManyRelatedManager
+django.contrib.auth.models.Permission_ManyRelatedManager
+django.contrib.auth.models.User_ManyRelatedManager
+
```

Comment:
These are only here since the stubs includes `= ManyToManyField(...)` in the following places:

https://github.com/typeddjango/django-stubs/blob/ec37d06ef630f7bc78a9df3a3cfd9030566655ad/django-stubs/contrib/auth/models.pyi#L34-L38

https://github.com/typeddjango/django-stubs/blob/ec37d06ef630f7bc78a9df3a3cfd9030566655ad/django-stubs/contrib/auth/models.pyi#L59-L62

Once they would move to a type hint form like: `field: ManyToManyField[..., ...]` they will disappear from the allowlist.